### ISSUE TITLE
Request: Suppress/Summarize/Clean Up API Response in Terminal Window

### ISSUE THREAD

**Issue by schmittx** [KraigM/homebridge-wink#22]
When a call to the Wink API is made, the entire results are shown in the terminal window. It'd be nice to suppress/hide these or just dump them to a file as it makes it hard to follow what Homebridge is doing in Terminal. Any thoughts?

**Comment by msroest** [comment]
This should be fixed by 1.0.3 that updated the version of wink-js.

**Comment by schmittx** [comment]
Yep, was running 1.0.2, confirmed that it's fixed in 1.0.3, thanks!

**Comment by KraigM** [comment]
glad to hear. thanks @msroest
